lib/errors.js

```diff
@@ -0,0 +1,8 @@
+function IssWhitelistError(message, params) {
+  Error.captureStackTrace(this, this.constructor)
+  this.name = 'IssWhitelistError'
+  this.message = message
+  if (params) this.params = params
+}
+IssWhitelistError.prototype = Object.create(Error.prototype)
+module.exports.IssWhitelistError = IssWhitelistError
```

Comment:
It looks like this new file is what's causing the CI builds to fail b/c we don't have 100% test coverage anymore.